docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,84 @@
    "id": "840ea662",
```

Comment:
<p>In classification -&gt; For classification</p><p>separately</p><p>"and if a single class doesn't pass the" -&gt; 'and if there is a class that doesn't pass the"</p><p> </p><p>Let's add a condition to the check and see what happens when it fails:</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>